pyproject.toml

```diff
@@ -1,10 +1,157 @@
+readme = "README.md"
+license = { text = "BSD" }
+requires-python = ">=3.8"
+authors = [
+    { name = "Philipp Rudiger", email = "developers@holoviz.org" },
+]
+maintainers = [
+    { name = "HoloViz developers", email = "developers@holoviz.org" },
+]
+classifiers = [
+        "License :: OSI Approved :: BSD License",
+        "Development Status :: 5 - Production/Stable",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
+        "Programming Language :: Python :: 3.12",
+        "Operating System :: OS Independent",
+        "Intended Audience :: Science/Research",
+        "Intended Audience :: Developers",
+        "Natural Language :: English",
+        "Topic :: Scientific/Engineering",
+        "Topic :: Software Development :: Libraries",
+]
+dependencies = [
+    "bokeh >=1.0.0",
+    "colorcet >=2",
+    "holoviews >=1.11.0",
+    "numpy >=1.15",
+    "packaging",
+    "pandas",
+    "panel >=0.11.0",
+    "param >=1.12.0,<3.0",
+]
+
+[project.entry-points."pandas_plotting_backends"]
+holoviews = "hvplot:plotting"
+
+[project.urls]
+Homepage = "https://hvplot.holoviz.org"
+Source = "http://github.com/holoviz/hvplot"
+HoloViz = "https://holoviz.org/"
+
+[project.optional-dependencies]
+tests-core = [
+    "codecov",
+    "dask[dataframe]",
+    "flake8",
+    "ipywidgets",
+    "matplotlib",
+    "parameterized",
+    "plotly",
+    "pooch",
+    "pre-commit",
+    "pytest-cov",
+    "pytest",
+    "scipy",
+    "xarray",
+]
+# Optional tests dependencies, i.e. one should be able
+# to run and pass the test suite without installing any
+# of those.
+tests = [
+    "hvplot[tests-core]",
+    "fugue",
+    "ibis-framework[duckdb]",  # ibis-duckdb on conda
+    "polars",
+]
+geo = [
+    "cartopy",
+    "fiona",
+    "geopandas",
+    "geoviews >=1.9.0",
+    "pyproj",
+    "rasterio",
+    "rioxarray",
+    "spatialpandas >=0.4.3",
+]
+# graphviz is difficult to install with pip, ok with conda.
+graphviz = [
+    "pygraphviz",
+]
+# Dependencies required to run the notebooks
+examples = [
+    "dask[dataframe] >=2021.3.0",
+    "datashader >=0.6.5",
+    "fugue",
+    "ibis-framework[duckdb]",  # ibis-duckdb on conda
+    "intake-parquet >=0.2.3",
+    "intake-xarray >=0.5.0",
+    "intake >=0.6.5,<2.0.0",
+    "ipywidgets",
+    "networkx >=2.6.3",
+    "matplotlib",
+    "notebook >=5.4",
+    "numba >=0.51.0",
+    "pillow >=8.2.0",
+    "plotly",
+    "polars",
+    "pooch >=1.6.0",
+    "s3fs >=2022.1.0",
+    "scikit-image >=0.17.2",
+    "scipy >=1.5.3",
+    "selenium >=3.141.0",
+    "streamz >=0.3.0",
+    "xarray >=0.18.2",
+]
+tests-nb = [
+    "pytest-xdist",
+    "nbval",
+]
+# Run the example tests by installing examples_tests together with tests
+examples-tests = [
+    "hvplot[examples]",
```

Comment:
I am curious: does this work if I change a package in examples?

Reply:
Yes.